user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -0,0 +1,227 @@
+   * [LCA inspiration](#lca-inspiration)
+
+<!-- TOC end -->
+
+## Colors / Themes
+
+- when starting to work on a app it essential to start from a good **theme**. If you like the look and feel of certain app duplicate it and start tuning it for your own app. For more information on how customize a theme, see [Configuring the dashboard layout](xref:Configuring_the_dashboard_layout).
+- when defining a background color for pages and panels (as part of the theme), never use full white (#FFFFFF) or full black (#000000). Some good defaults below:
+
+  |☀️  Light background |🌙 Dark background|
+  |------------------|----------------|
+  |#F2F2F2           |#1E232F         |
+- make sure your theme has a component style with a transparent background. Use this component style for titles, images, buttons.
+  ![Transparent Theme](~/user-guide/images/LCA_Style_Guide_Theme_Transparent.png)
+- also make a component style with a background color which differs from the page/panel background, this way they can easily be distinguish from the background. This can be used for query filters, tables, timelines, ... (anything but titles, images, buttons). Choose a slightly lighter color then the page/panel background, some good defaults below:
+  
+  |☀️  Light background |🌙 Dark background|
+  |------------------|----------------|
+  |#FDFDFD           |#282F3E         |
+- Make sure when picking colors, that there is enough contrast on text and icons in comparison with the background. This will greatly improve readability for the end user.
+
+  |👍 Good |👎 Bad |
+  |------|-----|
+  |![Good Contrast](~/user-guide/images/LCA_Style_Guide_Contrast_Good.png)|![Bad Contrast](~/user-guide/images/LCA_Style_Guide_Contrast_Bad.png)|
+- Same thing applies when picking a color for the app itself. If you choose an app color, the app will determine itself what the color of the text is based on the chosen background color. Be sure to pick a color which results in enough contrast between the text and background color.  
+
+  |👍 Good |👎 Bad |
+  |------|-----|
+  |![Good App Color](~/user-guide/images/LCA_Style_Guide_App_Color_Good.png)|![Bad App Color](~/user-guide/images/LCA_Style_Guide_App_Color_Bad.png)|
+- Limit the number of colors that you use in an app, don't make it too colorfull nor too monotone.
+- Picking good colors to start from is often a challenge, one way to approach it is to pick some colours from a layout you find online, like this sample below: 
+  ![Sample UI](~/user-guide/images/LCA_Style_Guide_Sample_UI.webp)
+
+## Navigation
+
+In order to have consistent navigation it is recommended to stick to the following order:
+
+1. Sidebar : main navigation should be the different pages defined in the app.
+1. Menubar : use this for navigation within a page, this can open up hidden page. In this case it make sense to change the title of each page to show where you are. 
+
+> [!CAUTION]
+> It is generally **not** advised to use buttons for navigation.
+
+## Titles
+
```

Comment:
Not sure if this is the right place, but can we add a guideline saying that they don't need to add customer logo's to apps? [https://teams.microsoft.com/l/message/19:c25cd70ae090429b922918a7adb2a892@thread.skype/1716390565470?tenantId=5f175691-8d1c-4932-b7c8-ce990839ac40&groupId=5581e7e5-76a8-425f-8027-1b17655dd05d&parentMessageId=1715936756799&teamName=Expert%20Hub%20-%20Deploying%20Solutions&channelName=05.%20ECS%20-%20UI%20UX%20Design&createdTime=1716390565470](Deploying solutions conversation)

Reply:
I have added this under a seperate section called #images